Experience/CSS/styles.css

```diff
@@ -0,0 +1,68 @@
+body {
```

Comment:
@zdburnett Try to keep your styles in one stylesheet and let all of the pages link to it.

Reply:
I combined styles sheets into a single one, and added a bio class to all elements in the html for bio and an exp class to all elements in the html experience. I then added a class to my style sheets to keep the styles from getting mixed up on the pages and relinked the html files to refer to the new css sheet.